### DISCUSSION TITLE
Windows port

### DISCUSSION THREAD

**Discussion by KAMiKAZOW** [organicmaps/organicmaps#373] [Ideas]
A Windows port for Surface and similar tablets would be nice

**Comment by rtsisyk** [comment]
You can try to build Qt version on Windows: https://github.com/omapsapp/omapsapp/blob/master/docs/INSTALL.md#desktop

**Comment by NathnaNeko** [comment]
How long do we need to wait for the "finished" package for Windows?

**Reply by KAMiKAZOW** [reply]
Considering that apparently nobody with the required skillset it working on that, I guess better don't hold your breath.

**Reply by biodranik** [reply]
If someone with engineering background starts this quest, we may help with fixing compile errors.
